nix/marlowe-cardano/deploy/nomadTasks.nix

```diff
@@ -1,6 +1,6 @@
 { inputs }:
 let
-  oci-images = inputs.self.oci-images.__iogx__;
+  oci-images = inputs.self.oci-images;
```

Comment:
Same here, make sure that this reflects the latest `deploy/nomadTasks.nix`. If I were you, I'd actually rename that file to this and port the differences in the `__iogx__` version rather than renaming the `__iogx__` version, they have gotten out of sync.

Reply:
Yes, this is much easier to review, thank you!